tests/Dockerfile.fedora

```diff
@@ -1,3 +1,5 @@
+ARG PREFIX=/usr/local
```

Comment:
This needs to go after the FROM instruction (FROM resets all ARGs).

Reply:
Mm, I was looking at https://docs.docker.com/build/building/variables/#scoping and quite possibly getting confused by the "wrong" example at the start where it requires separately "consuming" the ARG in further stages. I tried a lot of variants because much behavior of this didn't seem to make any sense to me at all, and got tired of waiting for the downloads, but lets try again with this then.

Reply:
Ah, this could be a Dockerfile specific thign then. I was looking solely at the Containerfile specification (which is used by Podman).

Reply:
We'll probably drop support for Docker anyway, and only use Podman in the future, so if this turns out to be a difference, it's probably not worth maintaining compatibility with.

Reply:
Oh, but now I realize, *this* is fine (to use ARG at the very beginning of the file). It's the second use of FROM that's confusing (see below).

Reply:
Okay, axed the whole dockerfile PREFIX thing, problem solved :smile: